portfolio/src/main/webapp/index.html

```diff
@@ -2,35 +2,62 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Trirong">
-    <link rel="stylesheet" href="style.css">
+    <title>Personal Portfolio</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Serif&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-        
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet">
+    
+    
+    <link href="style.css" rel="stylesheet" type="text/css" />
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>Tasha Ogoti's Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <div>
-        <img src="/images/blkwoman.jpg"/>
-        <p >My name is Tasha Ogoti.
-            <br> A junior student studying Computer Science at the University of Missouri - Columbia.
-            <br> Born and raised in Nairobi, Kenya.
-        </p>
-
+    <div  class="mainIntro">
+      <div class="navbar">
+        <img src="images/logo.png" class="logo" alt="myLogo" onclick = "window.location.assign(contactList.html)">
```

Comment:
In this page, clicking this image loads the contact list page, but in a few other pages, this behavior doesn't look like it exists. Let's make sure that the behavior of this image is consistent across all pages.